app/helpers/application_helper.rb

```diff
@@ -78,6 +78,16 @@ def admin_date(date)
     return "#{exact_date} (#{ago_text})"
   end
 
+  def read_asset_file(asset_name)
+    assets = Rails.application.assets_manifest.find_sources(asset_name)
+    if assets.first
+      assets.first.force_encoding('utf-8')
+    else
+      raise(Sprockets::FileNotFound,
```

Comment:
Always omit parentheses for Methods that have "keyword" status in Ruby:

https://github.com/bbatsov/ruby-style-guide#method-invocation-parens